src/kibana/plugins/markdown_vis/markdown_vis.less

```diff
@@ -5,3 +5,40 @@
   padding: 1em;
   width: 100%;
 }
+
+.vis-editor {
+
+  &.vis-type-markdown {
+
+    .vis-editor-config {
+      .flex-parent();
+
+      > ul {
+        .flex-parent();
+        .flex(1, 1, auto);
+
+        li {
+          .flex-parent();
+          .flex(1, 1, auto);
+        }
+      }
+    }
+
+    .visualization-options {
+      .flex-parent();
+      .flex(1, 1, auto);
+    }
+
+    .markdown-vis-options {
+      .flex-parent();
+      .flex(1, 1, auto);
+
+      textarea {
```

Comment:
Can you disable the little handle thingy at the bottom of the textbox? Its just breaks things.

![image](https://cloud.githubusercontent.com/assets/1329312/7737914/00053e96-ff04-11e4-98a3-9b85caf265b6.png)